step.yml

```diff
@@ -124,6 +124,16 @@ inputs:
       value_options:
       - "yes"
       - "no"
+  - auto_merge: "yes"
```

Comment:
This name seems a bit confusing to me because something is either manual or automatic but not both. We need a better name for this flag. What about `checkout_merged_state`?

Reply:
Yep I wasn't too keep on the name either